client/src/js/components/bhFundingSourceSelect/bhFundingSourceSelect.js

```diff
@@ -0,0 +1,55 @@
+angular.module('bhima.components')
+  .component('bhFundingSourceSelect', {
+    templateUrl : 'js/components/bhFundingSourceSelect/bhFundingSourceSelect.html',
+    controller  : FundingSourceSelectController,
+    transclude  : true,
+    bindings    : {
+      fundingSourceUuid : '<',
```

Comment:
If this parameter is optional (and it looks like it is on line 35), I would put a `?` on it.